Loop/Managers/DeviceDataManager.swift

```diff
@@ -1365,8 +1371,14 @@ extension DeviceDataManager {
             log.default("Remote Notification: Enacting bolus entry: %{public}@", String(describing: bolusAmount))
             
             //Remote bolus requires validation from its remote source
```

Comment:
Capture the error from the validation service to give the user more information on why something failed and how to fix it.